about.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/mobile.css" />
+    <link 
+    media="screen and (min-width: 768px)"
+      rel="stylesheet"
+      href="./styles/desktop.css"
+    >
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+
+    <title>Africa Tech Innovation Tour</title>
+  </head>
+  <body>
+    <header>
+      <nav class="container">
```

Comment:
Please make sure you are following the design. Here you need to remove the logo and should have transparent background to show the image on the background. Check the differences below.

<img width="387" alt="Screenshot 2023-01-08 at 2 00 40 AM" src="https://user-images.githubusercontent.com/25887253/211173406-da5a84c5-bdb5-4a46-a15e-fd6dd54df36e.png">
<img width="833" alt="Screenshot 2023-01-08 at 2 01 12 AM" src="https://user-images.githubusercontent.com/25887253/211173407-0899fb76-0b93-42ae-85c4-bbb630a3f4cf.png">